liblangutil/Token.h

```diff
@@ -297,6 +297,7 @@ namespace TokenTraits
 		op == Token::Add || op == Token::Mul || op == Token::Equal || op == Token::NotEqual; }
 	constexpr bool isArithmeticOp(Token op) { return Token::Add <= op && op <= Token::Exp; }
 	constexpr bool isCompareOp(Token op) { return Token::Equal <= op && op <= Token::GreaterThanOrEqual; }
+	constexpr bool isPragmaOp(Token op) { return (Token::LessThan <= op && op <= Token::GreaterThanOrEqual) || op == Token::BitXor; }
```

Comment:
I think that this should also include the = token. Additionally, the naming of the method could be something like isAllowedInPragma or similar.

Reply:
Probably, this function would never be used anywhere else... in that case, it would be better to not have it here and just use `isCompareOp` combined with the equality check for `Token::BitXor`.